msgraph/models.go

```diff
@@ -1144,6 +1144,7 @@ type User struct {
 	UsageLocation                   *StringNullWhenEmpty     `json:"usageLocation,omitempty"`
 	UserPrincipalName               *string                  `json:"userPrincipalName,omitempty"`
 	UserType                        *string                  `json:"userType,omitempty"`
+	EmployeeOrgData                 *EmployeeOrgData         `json:"employeeOrgData,omitempty"`
```

Comment:
Thanks @michaljirman! Could we move this up to maintain sorting of the field names?

Reply:
done